ceph-osd/tests/target.py

```diff
@@ -771,9 +771,16 @@ def tearDown(self):
 
         This ensures that the environment is ready for the next tests.
         """
-        zaza_model.run_action_on_units([self.TESTED_UNIT, ], 'start',
-                                       action_params={'osds': 'all'},
-                                       raise_on_failure=True)
+        try:
+            action_params = {'osds': 'all'}
+            print(f"Running {self.TESTED_UNIT} with params: {action_params}")
+            zaza_model.run_action_on_units([self.TESTED_UNIT, ], 'start',
+                                           action_params=action_params,
+                                           raise_on_failure=True)
+        except Exception as e:
+            print(f"Error running action on {self.TESTED_UNIT}: {e}")
+            available_osds = zaza_model.get_units('ceph-osd')
+            print(f"Available OSDs: {available_osds}")
```

Comment:
Why is this catch-all necessary?

Also, in the context of functests please use the logging module (e.g. logging.info() or .warning()) instead of printing.